pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto

```diff
@@ -668,6 +668,7 @@ enum ContainerState {
     CONTAINER_RUNNING = 1;
     CONTAINER_EXITED  = 2;
     CONTAINER_UNKNOWN = 3;
+    CONTAINER_OOMKILLED = 4;
```

Comment:
@Random-Liu I think it could be 137 but that can't be differentiated from SIGKILL. If we rely on Reason string, then we should strongly type it or at minimum document it in the API, so runtime implementations know they are supposed to set it without looking at kube code.